lib/encrypted.ex

```diff
@@ -0,0 +1,23 @@
+defmodule Fields.Encrypted do
+  alias Fields.AES
+
+  @behaviour Ecto.Type
+  def type, do: :binary
+
+  def cast(value) do
+    {:ok, to_string(value)}
+  end
+
+  def dump(value) do
+    ciphertext = value |> to_string |> AES.encrypt()
+    {:ok, ciphertext}
+  end
+
+  def load(value) do
+    {:ok, AES.decrypt(value)}
```

Comment:
where is the `AES.decrypt/1` function defined?

Reply:
That was previously defined, here: https://github.com/dwyl/fields/blob/master/lib/aes.ex#L61

Reply:
Thanks for reminding and linking to the definition. 👍